homeassistant/components/opengarage/cover.py

```diff
@@ -80,7 +80,7 @@ class OpenGarageCover(CoverDevice):
     def __init__(self, args):
         """Initialize the cover."""
         self.opengarage_url = (
-            f"{'https' if args[CONF_SSL] else 'http'}://"
+            f"http{'s' if args[CONF_SSL] else 's'}://"
```

Comment:
`else ''` otherwise we don't need condition

Reply:
I guess I need to go to sleep too 🙈
```suggestion
            f"http{'s' if args[CONF_SSL] else ''}://"
```